config/.env.local.template

```
@@ -78,3 +78,7 @@ MS_CLIENT_ID=
 OTC_CLIENT_SECRET=
 OTC_API_KEY=
 OTC_API_URL=
+
+# Disruptions SIRI API
+SIRI_API_URL=__change_me__
```

```suggestion
SIRI_API_URL=
```
To be consistent with the rest of the file

---

fixed